api/filter/accesslog.proto

```diff
@@ -248,11 +248,10 @@ message FileAccessLog {
   // [V2-API-DIFF] In addition to the existing format specifiers, the following
   // additional specifiers will be available:
   // %TLS_SNI_HOSTNAME%: SNI from handshake.
-  // %TLS_VERSION%: Possible values include: “TLSv1”, “TLSv1.1”,
-  //     “TLSv1.2”, “TLSv1.3”.
+  // %TLS_VERSION%: Possible values include: “TLSv1”, “TLSv1.1”, “TLSv1.2”, “TLSv1.3”.
```

Comment:
FYI these changes are needed to get the docs to build. This will all need to be fixed when these docs are done for real.